src/js/media-controller.js

```diff
@@ -239,19 +240,20 @@ class MediaController extends MediaContainer {
     // TODO: Can we just reset all state here?
     // Should hasPlayed refer to the media element or the controller?
     // i.e. the poster might re-show if not handled by the poster el
-    this.propagateMediaState(MediaUIAttributes.MEDIA_PAUSED, true);
+    this.propagateMediaState(MediaUIProps.MEDIA_PAUSED, true);
   }
 
   propagateMediaState(stateName, state) {
-    const previousState = this.getAttribute(stateName);
+    const attrName = stateName.toLowerCase();
+    const previousState = this.getAttribute(attrName);
```

Comment:
This still relies on the controller el attrs holding state? For some reason I thought we'd moved away from that with the controller updates.

Reply:
No, the media-controller is still the "source of truth" for checking state changes. That can change when we implement the separate media state model/store abstraction.

Reply:
Makes sense 👍